messaging/amqp/src/main/java/io/quarkus/ts/openshift/messaging/amqp/PriceProducer.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.ts.openshift.messaging.amqp;
+
+import io.reactivex.Flowable;
+import org.eclipse.microprofile.reactive.messaging.Outgoing;
+import org.jboss.logging.Logger;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+
+@ApplicationScoped
+public class PriceProducer {
+
+    private final Integer[] ints = new Integer[]{10, 20, 30, 40, 50, 60, 70, 80, 90, 100};
+    private final AtomicInteger p = new AtomicInteger(0);
+
+    private static final Logger LOG = Logger.getLogger(PriceProducer.class.getName());
+
+    @Outgoing("generated-price")
+    public Flowable<Integer> generate() {
+        LOG.info("generate fired...");
+        return Flowable.interval(1, TimeUnit.SECONDS)
```

Comment:
RxJava vs. Mutiny?

Reply:
@Ladicek  I don't know.  It seems Vert.x examples vs. Quarkus examples.    I suppose Mutiny then should be it?

Reply:
Probably.